packages/@guardian/source-react-components-development-kitchen/src/summary/styles.ts

```diff
@@ -12,7 +12,6 @@ export const wrapperStyles = (color: string): SerializedStyles => css`
 export const iconStyles = (color: string): SerializedStyles => css`
 	display: flex;
 	flex: 0 1 auto;
-	margin-top: 1px;
```

Comment:
This is causing issues with Chromatic diffs but seems the correct thing to do here.

Reply:
Doesn't work with Chromatic testing, have reverted to the old approach.